utils/mt5311/Makefile

```diff
@@ -0,0 +1,47 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mt5311
+PKG_VERSION:=238-2cb1304
+PKG_RELEASE:=1
+
+PKG_LICENSES:=AGPL-3.0-or-later
+PKG_LICENSE_FILES:=LICENSE
+PKG_URL:=https://github.com/jimdigriz/mt5311
+
+PKG_MAINTAINER:=jasle <jasle@riseup.net>
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/jimdigriz/mt5311.git
+PKG_SOURCE_VERSION:=2cb13045cd82474416e7f454b36f3a07b0de49d7
+PKG_MIRROR_HASH:=77ae6423f9aaaa11c562823df285105d9069fad0ad6a0380b8ac52e2357f3a1f
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/mt5311
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=Utilities for working with Metanoia/Proscend VDSL2 SFP Modems
+  URL:=https://github.com/jimdigriz/mt5311
+  DEPENDS:=+lua +luaposix +lua-struct
+endef
```

Comment:
We need also package description for this package.

Reply:
fixed.